src/components/map/Map.tsx

```diff
@@ -19,22 +16,13 @@ export default function Map() {
   const buckets = getBuckets(mapRecords.features);
 
   useEffect(() => {
-    if(state.country_prevalences.length > 0){
-      const prevalenceCountryDict: Record<string, AggregatedRecord> = state.country_prevalences.reduce((a: any, x: AggregatedRecord) => ({ ...a, [x.name]: x }), {});
+    if(state.estimate_grade_prevalences.length > 0){
```

Comment:
minor nit but consider calling this estimate_grade_metadata, cause technically it is more than just prevalences? I'll leave that up to your judgement tho